src/test/java/io/confluent/connect/elasticsearch/RetryUtilTest.java

```diff
@@ -45,6 +61,40 @@ public void computeNonRandomRetryTimes() {
     assertEquals(3200L, RetryUtil.computeRetryWaitTimeInMillis(5, 100L));
   }
 
+  @Test
+  public void testCallWithRetriesNoRetries() throws Exception {
+    MockTime mockClock = new MockTime();
+    long expectedTime = mockClock.milliseconds();
+
+    assertTrue(RetryUtil.callWithRetries("test", () -> testFunction(0), 3, 100, mockClock));
+    assertEquals(expectedTime, mockClock.milliseconds());
+  }
+
+  @Test
+  public void testCallWithRetriesSomeRetries() throws Exception {
```

Comment:
nit: add a unit-test that ensures wait time is always lesser than `MAX_RETRY_TIME_MS`?

Reply:
already tested by https://github.com/confluentinc/kafka-connect-elasticsearch/blob/master/src/test/java/io/confluent/connect/elasticsearch/RetryUtilTest.java#L57
and
https://github.com/confluentinc/kafka-connect-elasticsearch/blob/master/src/test/java/io/confluent/connect/elasticsearch/RetryUtilTest.java#L23-L46